backend/curriculum_tracking/tests/test_api_views.py

```diff
@@ -308,3 +308,32 @@ def verbose_instance_factory(self):
 
 #     def verbose_instance_factory(self):
 #         return factories.ContentItemOrderFactory()
+
+
+class WorkshopAttendanceViewsetTests(APITestCase, APITestCaseMixin):
+    LIST_URL_NAME = "workshopattendance-list"
+    SUPPRESS_TEST_POST_TO_CREATE = True
+
+    def verbose_instance_factory(self):
+        workshop = factories.ContentItemFactory(content_type=ContentItem.WORKSHOP)
+        workshop_attendance = factories.WorkshopAttendanceFactory(content_item=workshop)
+        content = workshop_attendance.content_item
+        content.topic_needs_review = False
+        content.save()
+        return workshop_attendance
+
+    def test_get_instance_permissions(self):
+
+        random_human = UserFactory(is_superuser=False, is_staff=False)
+        self.login(random_human)
+        url = self.get_list_url()
```

Comment:
This is a bit strange - your test name says it's about instance permissions, but you are using the `get_list_url` url with filters...

You aren't actually testing the permissions you created.

You should rather get a specific instance instead of a list of instances if you want to test your instance permissions.

Eg:   `url = f"{self.get_list_url()}/{workshop_attendance.id}"` 

If a user wants to fetch their own attendance instance then they should be able to